pom.xml

```diff
@@ -49,6 +51,43 @@
 				<type>pom</type>
 				<scope>import</scope>
 			</dependency>
+
+			<dependency>
+				<groupId>commons-logging</groupId>
+				<artifactId>commons-logging</artifactId>
+				<version>1.2</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-sleuth-core</artifactId>
+				<version>${spring-cloud-sleuth.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-starter-sleuth</artifactId>
+				<version>${spring-cloud-sleuth.version}</version>
+			</dependency>
+
+			<dependency>
+				<groupId>mysql</groupId>
+				<artifactId>mysql-connector-java</artifactId>
+				<version>5.1.42</version>
```

Comment:
Spring  Boot already has [dependency management on 5.1.44](https://github.com/spring-projects/spring-boot/blob/98f577e62c7bd9d95663b0c9c69cc0a4c2d8e788/spring-boot-dependencies/pom.xml#L137) for this one. I strongly recommend you not to deviate from the version that Spring Boot provides. If you have a good reason to do that, I am happy to hear what it is